Formula/s/sq.rb

```diff
@@ -0,0 +1,43 @@
+class Sq < Formula
+  desc "Data wrangler with jq-like query language"
+  homepage "https://sq.io"
+  url "https://github.com/neilotoole/sq/archive/refs/tags/v0.48.3.tar.gz"
+  sha256 "46e75e2db83a6cbc98b07dbcfb23de03fc41b2b2cbc7de7aaee0425cef4fb9bb"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  uses_from_macos "sqlite" => :test
+
+  def install
+    icu4c = Formula["icu4c"]
+    ENV.append "CFLAGS", "-I#{icu4c.opt_include}"
+    ENV.append "LDFLAGS", "-L#{icu4c.opt_lib}"
+    pkg = "github.com/neilotoole/sq/cli/buildinfo"
+    ldflags = %W[
+      -s -w
+      -X #{pkg}.Version=v#{version}
+      -X #{pkg}.Commit=RELEASE
+      -X #{pkg}.Timestamp=#{Time.now.utc.strftime("%Y-%m-%dT%H:%M:%SZ")}
+    ]
+    tags = %w[
+      netgo sqlite_vtable sqlite_stat4 sqlite_fts5 sqlite_introspect
+      sqlite_json sqlite_math_functions
+    ]
+    system "go", "build", *std_go_args(ldflags:), "-tags", tags.join(" ")
+    generate_completions_from_executable(bin/"sq", "completion")
+    (man1/"sq.1").write Utils.safe_popen_read(bin/"sq", "man")
+  end
+
+  test do
+    (testpath/"test.sql").write <<~EOS
+      create table t(a text, b integer);
+      insert into t values ('hello',1),('there',42);
+    EOS
+    system "sqlite3 test.db < test.sql"
+    out1 = shell_output("#{bin}/sq add --active --handle @tst test.db")
+    assert_equal %w[@tst sqlite3 test.db], out1.strip.split(/\s+/)
+    out2 = shell_output("#{bin}/sq '@tst.t | .b' </dev/null 2>&1")
```

Comment:
```suggestion
    out2 = pipe_output("#{bin}/sq '@tst.t | .b' 2>&1", "")
```
Might work here.

Reply:
Nope, sorry. With `pipe_output`,  even an interactive `brew test` abends on every platform with `sq: EOF`. That `/dev/null` redirection is needed for some reason.